packages/dashboard/src/layout/LayoutUtils.ts

```diff
@@ -597,8 +601,8 @@ class LayoutUtils {
         // container property exists on a contentItem if contentItem is a component,
         // however, this is not included in the types for a contentitem, thus the casting
         ((oldContentItem as unknown) as {
-          container: { close: () => void };
-        }).container.close();
+          container: { close: (isOverwrite?: boolean) => void };
```

Comment:
A cleaner way to write this whole block without the casting/comment is to use the `isComponent` type predicate that is defined in `AbstractContentItem`.
Combined with my other comment about defining `CloseOptions`, we shouldn't need any casting and should just be able to do:
```
if (isComponent(oldContentItem)) {
  oldContentItem.container.close(closeOptions);
}
```